src/tools/web/webd/src/index.js

```diff
@@ -31,13 +31,13 @@ getVersions()
       error(`are you sure you have libelektra and kdb installed?`);
       process.exit(1);
     } else {
-      const { major, minor, patch } = versions.elektra;
-      const versionSupported = major >= 0 && minor >= 8 && patch >= 23;
+      const { major, minor, micro } = versions.elektra;
+      const versionSupported = major >= 0 && minor >= 9 && micro >= 0;
       if (!versionSupported) {
         error(
           `you are running an old libelektra version, which is not supported`
         );
-        error(`please upgrade to libelektra 0.8.23 or higher`);
+        error(`please upgrade to libelektra 0.9.0 or higher`);
```

Comment:
I get this error message when running the elektra/web docker image (after building the image from your branch). Running kdb -V in the container I get 0.9.1 so it should not be a problem, I'm confused.

Reply:
@raphi011 did you look into how much effort it would be to also port webd to Go? It would be great to not have this mixture of different languages within a single tool.

Reply:
> I get this error message when running the elektra/web docker image (after building the image from your branch). Running kdb -V in the container I get 0.9.1 so it should not be a problem, I'm confused.

i fixed this in my branch `version.micro` was renamed to `version.patch`

> did you look into how much effort it would be to also port webd to Go? It would be great to not have this mixture of different languages within a single tool.

Not yet